internal/controller/apiextensions/composite/composed.go

```diff
@@ -352,13 +361,6 @@ func (r *APIDryRunRenderer) Render(ctx context.Context, cp resource.Composite, c
 	cd.SetName(name)
 	cd.SetNamespace(namespace)
```

Comment:
@ulucinar would this change mean that it is now impossible to patch over the `metadata.name` and `metadata.namespace` values? If so, I think this may be a breaking change that we are not willing to make. The existing behavior is desirable in cases where you want to provision a resource like a `ProviderConfig` with an identifying name that is used to group all the resources in a composition: https://github.com/upbound/platform-ref-aws/blob/63e724fc104ca1147c61787acc8a2891aced0393/cluster/eks/composition.yaml#L180

Reply:
Hi @hasheddan,
Yes it would prevent such use-cases, thanks for pointing to the example use-case.

Reply:
With the second commit, we now allow such patches.